### PR TITLE
Update experimental demo app

### PR DESCRIPTION
**Pull request by QuncCccccc**
This pr adds Switch and Checkbox to demo app.

Screen Shot 2022-09-09 at 5 25 49 PM

## Pre-launch Checklist

- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I read the [Contributors Guide].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.

### CODE REVIEW

File: experimental/material_3_demo/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 
-This app also includes new M3 components that haven't landed in the Flutter stable channel, such as IconButtons, Chips and TestFields. The app will be migrated to the non-experimental directory in the samples repo once all the components land in the stable branch. For more information about the components in the stable channel, please check the "material_3_demo" in the non-experimental directory.
+This app also includes new M3 components that haven't landed in the Flutter stable channel, such as IconButtons, Chips, TestFields, Switches and Checkboxes. The app will be migrated to the non-experimental directory in the samples repo once all the components land in the stable branch. For more information about the components in the stable channel, please check the "material_3_demo" in the non-experimental directory.
```
Comment (darrenaustin) [inline experimental/material_3_demo/README.md:5]: Typo: TextFields

File: experimental/material_3_demo/README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ This app also includes new M3 components that haven't landed in the Flutter stab
 <img src="https://user-images.githubusercontent.com/36861262/166511137-85dea8df-0017-4649-b913-14d4b7a17c2f.png" width="25" /> This button will bring up a pop-up menu that allows the user to change the base color used for the light and dark themes. This uses a new color seed feature to generate entire color schemes from a single color.
 
 ## Component Screen
-The default screen displays all the updated components in Material 3: AppBar, common Buttons, Floating Action Button(FAB), Chips, Card, Dialog, NavigationBar, and NavigationRail.
+The default screen displays all the updated components in Material 3: AppBar, common Buttons, Floating Action Button(FAB), Chips, Card, Checkbox, Dialog, NavigationBar, NavigationRail, TestFields and Switch.
```
Comment (darrenaustin) [inline experimental/material_3_demo/README.md:21]: Same here 😀

**Review by darrenaustin — APPROVED** [review]
LGTM. Couple of small typos. Also this is still a draft?

**Comment by QuncCccccc** [comment]
> LGTM. Couple of small typos. Also this is still a draft?

Marked as Ready to Review! Thanks:)

**Pull request merged**